.github/workflows/docker-publish.yaml

```diff
@@ -99,4 +99,16 @@ jobs:
           tags: |
             ${{ steps.login-ecr.outputs.registry }}/lakefs:${{ steps.version.outputs.tag }}
             treeverse/lakefs:${{ steps.version.outputs.tag }}
-            treeverse/lakefs:latest
\ No newline at end of file
+            treeverse/lakefs:latest
+            
+      - name: Build and push lakefs quickstart
```

Comment:
if we will get to the point where we will need to version quickstart images I prefer to keep it simple and think of alternative way to identify quickstart usage - arg? or just env var and not new image.